src/meta/src/manager/system_param/model.rs

```diff
@@ -12,41 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::collections::HashSet;
-
 use async_trait::async_trait;
+use risingwave_common::system_param::{system_params_from_kv, system_params_to_kv};
 use risingwave_pb::meta::SystemParams;
 
 use crate::model::{MetadataModelError, MetadataModelResult};
 use crate::storage::{MetaStore, Transaction};
 
-const SYSTEM_PARAM_CF_NAME: &str = "cf/system_params";
-
-const BARRIER_INTERVAL_MS_KEY: &str = "barrier_interval_ms";
-const CHECKPOINT_FREQUENCY_KEY: &str = "checkpoint_interval";
```

Comment:
This is changed. Is this expected? 🤣

https://github.com/risingwavelabs/risingwave-docs/pull/1002

Reply:
These hard-coded constants were only used for a few commits before I refactored the code to use macro to generate these keys (`const CHECKPOINT_FREQUENCY_KEY: &str = "checkpoint_interval";` seems to be a mistake here). I think it has been `checkpoint_frequency` after this commit.